blocks/variables.ts

```diff
@@ -142,29 +140,31 @@ const CUSTOM_CONTEXT_MENU_VARIABLE_GETTER_SETTER_MIXIN = {
 /**
  * Factory for callbacks for rename variable dropdown menu option
  * associated with a variable getter block.
- * @param {!Block} block The block with the variable to rename.
- * @return {!function()} A function that renames the variable.
+ * @returns A function that renames the variable.
```

Comment:
I think the @param tag should be preserved (just without a type declaration)—and a blank line should be added, for consistency with eslint rules (not sure why it's not complaining here):

```suggestion
 *
 * @param block The block with the variable to rename.
 * @returns A function that renames the variable.
```